File: backend/ruby_foundations/exercises/small_problems_101/easy_9/grade_book.rb
def get_grade(score1, score2, score3)
  average = (score1 + score2 + score3) / 3.to_f
  case average
  when (90..100)
    "A"
  when (80...90)
    "B"
  when (70...80)
    "C"
  when (60...70)
    "D"
  when (0...60)
    "F"
  end
end

puts get_grade(95, 90, 93) == "A"
puts get_grade(50, 50, 95) == "D"

File: frontend/javascript/exercises/easy_2/grade_book.js
function getGrade(score1, score2, score3) {
  var totalScores = score1 + score2 + score3;
  var averageScore = totalScores / 3;
  if (averageScore >= 90 && averageScore <= 100) {
    return 'A';
  } else if (averageScore >= 80 && averageScore < 90) {
    return 'B';
  } else if (averageScore >= 70 && averageScore < 80) {
    return 'C';
  } else if (averageScore >= 60 && averageScore < 70) {
    return 'D';
  } else {
    return 'F';
  }
}

console.log(getGrade(95, 90, 93)); // "A"
console.log(getGrade(50, 50, 95)); // "D"